src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+$text-color: #616070;
+$button-text-color: #00acdc;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+  padding-inline: 16px;
+  transition: all 300ms linear;
+
+  &:hover{
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 3px 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    margin-top: 40px;
+  }
+
+  &__code {
+    box-sizing: border-box;
+    line-height: 14px;
+    font-weight: 400;
+    font-size: 10px;
+    color: $text-color;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    box-sizing: border-box;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 16px;
+    align-items: center;
+    height: 16px;
+  }
+
+  &__reviews-value {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__price {
+    height: 18px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 24px;
+    align-items: center;
+
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $text-color;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: #060b35;
+    }
+  }
+
+  &__button {
+    text-transform: uppercase;
+    box-sizing: border-box;
+    width: 166px;
+    height: 40px;
+    background-color: $button-text-color;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    margin-block: 16px;
+    border-radius: 5px;
+    border: none;
+    transition: all 300ms linear;
+
+    &:hover {
+      border: solid 1px $button-text-color;
+      background-color: #fff;
+      color: $button-text-color;
+    }
```

Comment:
When hovering over the button, the border is added which increases the total width and height of the button by 2px (1px for each side). This may cause the layout to shift. To avoid this, you can set a transparent border for the normal state, and change the border color on hover.